app/db_utils.go

```diff
@@ -0,0 +1,160 @@
+package app
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/lib/pq"
```

Comment:
According to golint, **a blank import should be only in a main or test package, or have a comment justifying it**, and this [post](https://www.calhoun.io/why-we-import-sql-drivers-with-the-blank-identifier/), you should add a comment after it.